pkg/operation/types.go

```diff
@@ -72,6 +72,7 @@ type Operation struct {
 	APIServerAddress          string
 	APIServerHealthCheckToken string
 	SeedNamespaceObject       *corev1.Namespace
+	ClusterIdentity           *string
```

Comment:
Why is this a pointer?

Reply:
I prefer to be nil if no one set it, instead of "".

Reply:
But it will be always set in the shoot reconciliation, so how is it different from fields like `APIServerAddress` or `APIServerHealthCheckToken`

Reply:
You are right. I (🤡) was thinking for cases where others try to use it before it is set, and because it is not part of the `Botanist` init to panic if they try to dereference it or at least suggest that this value might not be set .

Reply:
Generally, this might make sense, but I don't think this use-case exists. The whole package is not build in a way that "others" could or should use it.  :-(